docs/terms/datasource.md

```diff
@@ -68,11 +63,9 @@ context = ge.get_context()
 datasource = context.get_datasource("my_datasource_name")
 ```
 
-<!---
+### How to create and configure
+
+Creating a Datasource is quick and easy, and can be done from the <TechnicalTag relative="../" tag="cli" text="CLI" /> or through Python code.  Configuring the Datasource may differ between backends, according to the given backend's requirements, but the process of creating one will remain the same.
```

Comment:
two thoughts:
- would it be valuable to include a high level overview of what this process would be? Otherwise saying that 'the process to create one will remain the same' feels a bit incomplete. maybe something like:
```
To create a new  Datasource through the CLI, run `great_expectations datasource new`.
To create a new Datasource through Python code, obtain a data context and call its `add_datasource` method.
```

- We can also create a new datasource directly through the YAML config file. Is this intentionally excluded here?

Reply:
I've added the proposed notes, as well as one stating that advanced users may create a new Datasource directly through a YAML config file.